plate_simulation/models/events.py

```diff
@@ -15,20 +15,58 @@
 from trimesh import Trimesh
 from trimesh.proximity import ProximityQuery
 
+from plate_simulation.models import EventMap
+
 
 # pylint: disable=too-few-public-methods
 
 
+def increment_name_list(name: str, name_list: list[str]) -> str:
```

Comment:
We have this function in geoh5py.shared.utils.find_unique_name

Reply:
Oh nice, was looking for it. Thanks.